Formula/img4tool.rb

```diff
@@ -36,6 +37,6 @@ def install
                           "--disable-silent-rules",
                           "--prefix=#{prefix}"
     system "make"
-    bin.install "img4tool/img4tool"
+    system "make", "install"
```

Comment:
When using --HEAD to install img4tool with this formula I got a binary with version 154, even though it has the same hash as 163. When installing without --HEAD all is fine. I suspect this line but I don't know why.

Reply:
That's strange; the binary on my device correctly reports itself as version 163 (well, version `0.163-c3df16cb998d5013aef092baeb71908475366182`, but it's close enough).
It's likely due to tihmstar's revision of the build/install process for his programs; it appears he's now using `.pc.in` files.
The way I got libipatcher to report itself as version 68 was by replacing `@VERSION_COMMIT_COUNT@` in `libipatcher.pc.in` with `68`; this is likely relevant.